Makefile.ci

```diff
@@ -24,6 +24,7 @@ CI_TARGETS= \
     ci-coquelicot \
     ci-corn \
     ci-cross_crypto \
+    ci-coq-library-undecidability \
```

Comment:
The part of the name after `ci-` must be a valid shell variable, ie use underscores instead of dashes
```suggestion
    ci-coq_library_undecidability \
```
other occurences must match (`project` in basic-overlay, the shell script name, the gitlab job name)